packages/api/src/background/FinchConnection.test.ts

```diff
@@ -84,6 +84,45 @@ describe('connections', () => {
 
       teardown();
     });
+    it('should send messages from the port to a passed handler', async () => {
+      // Setup connection start listening for connections
+      const connection = new FinchPortConnection({
+        messagePortName: 'test',
+        external: false,
+      });
+      const teardown = connection.onStart();
```

Comment:
question. is `teardown` better than something like `tearup`? i only ask because this is `onStart()` and to me this sounds like you're building up something.

Reply:
yeah, the `onStart` method returns a function that tearsdown the connection.